<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mapper_game</class>
 <widget class="QMainWindow" name="Mapper_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 500 13pt &quot;BM Jua&quot;;
background: rgb(255, 246, 233);
color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 20pt;
font: bold;
background: rgb(187, 226, 236);
border: 1px rgb(187, 226, 236);
border-radius: 12px;
letter-spacing:1px;</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="GroupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>541</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string>You have guessed</string>
    </property>
    <widget class="QLabel" name="Country_head">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Jua</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Country</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="country1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Continent_head">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Jua</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Continent</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Language_head">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Jua</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Language</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Border_head">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Jua</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Border</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="country2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="country3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="country4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="country5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="country10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>550</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="country9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="country6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="country8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="country7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent1">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>100</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent2">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>150</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent3">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>200</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent4">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>250</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent5">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>300</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent6">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>350</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent7">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>400</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent8">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>450</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent9">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>500</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="continent10">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>550</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang1">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>100</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang2">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>150</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang3">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>200</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang4">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>250</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang5">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>300</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang6">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>350</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang7">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>400</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang8">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>450</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang9">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>500</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lang10">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>550</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="border1">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>100</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border2">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>150</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border3">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>200</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border4">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>250</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border5">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>300</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border6">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>350</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border7">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>400</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border8">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>450</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border9">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>500</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border10">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>550</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="country_shape">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="guessShapecomboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
